content/1.7/back-office/international/locations/countries/add-edit-country.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Add edit country
+weight: 1
+---
+
+# Add edit country
+
+When adding or editing a country, the following fields / drop-down / toggles are displayed:
+
+ - **Country** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The field name is required at least in English (English).". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+The following tooltip is displayed: "Country name - Invalid characters: <>;=#{}".
+
+ - **ISO code** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).".
+
+ - **Call prefix** - This is a mandatory field. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)".
+ 
+ - **Default currency** - This is a drop-down field which allows to seect a currency from the ones that are enabled on the shop.
+ 
+ - **Zone** - This is a drop-down field with all the zones that can be selected. The following tooltip is displayed: "Geographical region.".
+ 
+ - **Does it need Zip/Postal code?** - Toggle switch for either "yes" or "no". By default is set to "yes".
+
+ - **Zip/Postal code format** - This is a mandatory field.
+The following helptext below is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank."
+
+ - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
```

Comment:
```suggestion
 - **Address format** - Firstly, a box is displayed with the following default values: firstname lastname, company, vat_number, address1, address2, postcode city, Country:name, phone. On the right, a help text is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
```

Reply:
What happens if I save and if a parameter already existing has been added many times in the address format ?

Reply:
@marionf It adds for me without any issues, but not sure if it's a bug or not